web/src/engine/osk/src/visualKeyboard.ts

```diff
@@ -1604,6 +1606,11 @@ export default class VisualKeyboard extends EventEmitter<EventMap> implements Ke
     // the page.
     kbdObj.appendStyleSheet();
 
+    // Unset the width + height we used thus far; this method's consumer may choose to rescale
+    // the returned element.  If so, we don't want to use our outdated value by mistake.
+    delete kbdObj._width;
+    delete kbdObj._height;
```

Comment:
Whlie we _could_ in theory call `kbdObj.setSize()` (no parameters) to re-enable default scaling, this would also remove the keyboard size styling we normally apply within the method.

Note that `buildKeyboardDocumentation` explicitly takes a `height` parameter to use for this styling.  It should still exist after the function completes, even if we allow it to be overwritten and adjusted afterward.  Should that overwrite _not_ happen, we need the keyboard to have some value established for its `height` styling.

Reply:
Could you add your comment here in some form to the source?

Reply:
So, _extend_ the comment that's already there?  I take it the explanation here, while more verbose, is overall clearer.